.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+## Summary
```

Comment:
Should we also have a template for shiproom-approval-needed PRs? I always go and copy/paste the sections from a recent one, which is annoying :)

Reply:
Great idea! It looks like we'll have to access it by manually typing in the url like so (or bookmarking it): https://github.com/microsoft/msbuild/pull/new?template=shiproom.md